scrapi/harvesters/crossref.py

```diff
@@ -40,23 +43,24 @@ def copy_to_unicode(self, element):
             return unicode(element, encoding=encoding)
 
     def harvest(self, days_back=0):
-        base_url = 'http://api.crossref.org/v1/works?filter=from-pub-date:{},until-pub-date:{}&rows=1000'
         start_date = date.today() - timedelta(days_back)
-        url = base_url.format(str(start_date), str(date.today()))
-        data = requests.get(url)
-        doc = data.json()
-
-        records = doc['message']['items']
+        base_url = 'http://api.crossref.org/v1/works?filter=from-pub-date:{},until-pub-date:{}&rows={{}}&offset={{}}'.format(str(start_date), str(date.today()))
```

Comment:
```furl('http://api.crossref.org/v1/works/').add({'filters': 'from-pub-date:{},until-pub-date:{}'.format(start_date, date.today())``` is what it would actually look like